options/options.js

```diff
@@ -10,20 +12,45 @@ function display_status(message) {
 // Saves options to chrome.storage
 function save_options() {
   var theme = document.getElementById("theme").value;
+  var exceptions = document.getElementById("exceptions").value
+      .split('\n')
+      .map(Function.prototype.call, String.prototype.trim);
+  var exceptionsMap = {};
+  exceptions.forEach(function(item) {
+    if (item) {
+      exceptionsMap[item] = true;
+    }
+  })
   chrome.storage.local.set({
     theme: theme,
+    exceptions: exceptionsMap,
   }, display_status.bind(this, "Option saved!"));
 }
 
+// Save the option if the user stops writing for a second
+function save_options_with_delay() {
+  if (timeout) {
+    window.clearTimeout(timeout);
+  }
+  timeout = window.setTimeout(save_options, 1000);
+}
+
 // Restores select box and checkbox state using the preferences
 // stored in chrome.storage.
 function restore_options() {
   chrome.storage.local.get({
-    theme: "default"
+    theme: "default",
+    exceptions: ""
   }, function(items) {
     document.getElementById("theme").value = items.theme;
+    var exceptionList = document.getElementById("exceptions");
+    for (var exception in items.exceptions) {
+        exceptionList.value += exception + '\n';
```

Comment:
That is avoided by:
`if (item)` (on line 20)
An empty string will be false, and thus is not added.